packages/server/src/service/letter/followUp.ts

```diff
@@ -0,0 +1,110 @@
+import marked from 'marked'
```

Comment:
Can you add a screenshot of an example result of this?

Reply:
![Screenshot_20201007_154801](https://user-images.githubusercontent.com/31702297/95374620-17c4f100-08ce-11eb-988d-739dbe2e39b1.png)

P.S.: Unregistered Voter is actually the name I've used to complete this sign up, it was from the time I was implementing Alloy features.